src/service/viewer-impl.js

```diff
@@ -302,12 +302,16 @@ export class ViewerImpl {
     if (!isEmbedded) {
       return null;
     }
+    const timeoutMessage = 'initMessagingChannel timeout';
     return Services.timerFor(this.win)
-      .timeoutPromise(20000, messagingPromise, 'initMessagingChannel')
+      .timeoutPromise(20000, messagingPromise, timeoutMessage)
       .catch(reason => {
         const error = getChannelError(
           /** @type {!Error|string|undefined} */ (reason)
         );
+        if (error && error.message === timeoutMessage) {
+          error.expected = true;
```

Comment:
This leaks implementation detail of error.js. I think `createExpectedError` will wrap a given error object.

Reply:
Good catch; I was thinking of that method as a way to create an error, I completely forgot it also handles modifying existing ones! Done.